.github/workflows/documentation.yml

```diff
@@ -0,0 +1,34 @@
+name: deploy documentation
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+    steps:
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '8.1'
+
+      - name: Checkout Code
+        uses: actions/checkout@v2
+
+      - name: Build documentation with doctum
+        run: ./scripts/run_doctum_docs_on_github.sh
+
+      - name: Deploy
+        uses: reggionick/s3-deploy@v3
+        with:
+          folder: doctum-output/build/
+          bucket: rokka-io
+          bucket-region: eu-central-1
+          dist-id: ${{ secrets.CLOUDFRONT_DISTRIBUTION_ID }}
+          invalidation: /client-php-api/
+          delete-removed: false
```

Comment:
i suggest we merge with this set to false and check if the files are uploaded to the right place. does the access key define the target folder? or how does the system know where in the bucket to place the files?

Reply:
Didn't fine an option to define a target folder, but the `build` folder has the right structure. I'd leave this to "false" all the time, isn't a big deal, if we have "old" files on S3 (there won't be many anyway, I guess)

Reply:
ah, i see. yeah then this must be false or we wipe the rest of the website :scream: